### ISSUE TITLE
[FEATURE REQUEST] https://teaas.fyi/ api for any emoji.

### ISSUE THREAD

**Issue by ClashTheBunny** [jackellenberger/emojme#64]
Have the ability to `onFire(emoji)` or to `dealWithIt(emoji)`!

**Comment by Yhprum** [comment]
I took a gander at this - teaas doesn't seem to have an api that's publicly available/usable. It might be possible to scrape html to get the response image, but maybe implementing fire using some gif library would be easier.

**Comment by ClashTheBunny** [comment]
Yeah, I wonder how hard it would be to extend https://github.com/scotchfield/party-party-party or something...

**Comment by jackellenberger** [comment]
the code for teaas is open source as well at https://gitlab.com/wjr1985/teaas (slash https://gitlab.com/wjr1985/teaas_puma_example) (where i have contributed) and on github (https://github.com/wjr1985/teaas, though that might be just a clone, unclear which is a clone of which). 

I think if you wanted to API ify it that would be awesome. something something protobufs idk
